src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -120,8 +119,26 @@ internal sealed class ChannelExecutorConfigurator : ExecutorServiceConfigurator
     {
         public ChannelExecutorConfigurator(Config config, IDispatcherPrerequisites prerequisites) : base(config, prerequisites)
         {
-            var cfg = config.GetConfig("channel-executor");
-            Priority = (TaskSchedulerPriority)Enum.Parse(typeof(TaskSchedulerPriority), cfg.GetString("priority", "normal"), true);
+            var priorityName = config.GetString("channel-executor.priority", "None") ?? "None";
+            Priority = (TaskSchedulerPriority)Enum.Parse(typeof(TaskSchedulerPriority), priorityName, true);
+            if (Priority == TaskSchedulerPriority.None)
+            {
+                var dispatcherName = config.Root.GetObject().GetKey("name").ToString();
+                switch (dispatcherName)
+                {
+                    case "internal-dispatcher":
+                    case "default-remote-dispatcher":
+                        Priority = TaskSchedulerPriority.High;
```

Comment:
These default values should be baked in the default HOCON config, not in code; i.e. they have to be transparent to the user, user should only read the config file to figure out the default value of a setting, not dig through the source code.

Reply:
they are the default in the config.
But the testkit does not load the default config